redis-persistence/build.gradle

```diff
@@ -8,7 +8,6 @@ dependencies {
  	compile "com.netflix.dyno-queues:dyno-queues-redis:${revDynoQueues}"
 	compile "org.elasticsearch:elasticsearch:${revElasticSearch2}"
 
-	//In memory redis for unit testing
- 	testCompile "org.rarefiedredis.redis:redis-java:${revRarefiedRedis}"
-
+    //In memory
+    compile "org.rarefiedredis.redis:redis-java:${revRarefiedRedis}"
```

Comment:
what is reason of changing from testCompile to compile?

Reply:
The mock that is used for local development requires this at compile time.

`com.netflix.conductor.jedis.JedisMock`

You can see the corresponding removal from the server module compile dependencies here: https://github.com/Netflix/conductor/pull/550/files#diff-2af3da569e27d5fad9d8c14613c35a66L42